lib/mix/prepare_assets.ex

```diff
@@ -115,6 +116,23 @@ defmodule Mix.Tasks.Antikythera.PrepareAssets do
     end
   end
 
+  defp audit_vulnerability!(env) do
```

Comment:
How about using `audit_vulnerability` as this function name until `raise` has been used?

Reply:
fixed ecbe8a57c6eaa981aee08f221dddcb9afad24aee